test/benchee/benchmark/runner_test.exs

```diff
@@ -121,6 +121,25 @@ defmodule Benchee.Benchmark.RunnerTest do
       assert length(memory_usages) > 0
     end
 
+    test "measures the reduction count of a scenario" do
+      suite =
+        test_suite(%Suite{
+          configuration: %{time: 60_000, warmup: 10_000, reduction_time: 2_000_000}
```

Comment:
do we need such a long time? Normally we make do with much less time don't we?

Reply:
So I don't know why, but this fails in CI unless we run it for some time. There could be some bug there, but it seems related to CI only since I can't reproduce this on my local machine at all.

Reply:
Interesting, what does it fail with? :thinking:  because that's... eh is that 2 seconds? That'd be pretty rough.

Reply:
I figured this out and fixed it!